index.md

```diff
@@ -29,7 +29,7 @@ This code is not exhaustive or complete. It serves to distill our common underst
 
 ### Diversity Statement
 
-We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is equal. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
+We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is worthy of respect. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
```

Comment:
I can see that point and agree.
In which case I'd still stick with equality, though, but with a qualification along the lines of your [suggestion above](https://github.com/todogroup/opencodeofconduct/pull/9#discussion_r35242081), e.g. something like "should be treated equally with respect" etc.